src/index.html

```diff
@@ -8,6 +8,54 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <main class="container">
+      <section class="stars stars--0">
```

Comment:
it looks not like a section. Just use div here. Apply for other cases as well
```suggestion
      <div class="stars stars--0">
```